openpype/hosts/maya/api/pipeline.py

```diff
@@ -625,10 +629,26 @@ def _on_show_inventory():
     check_lock_on_current_file()
 
 
+def _auto_build_first_workfile():
+    if workfile_template_builder.should_build_first_workfile():
+        build_workfile_template()
+
+
 def on_new():
     """Set project resolution and fps when create a new file"""
     log.info("Running callback on new..")
     with lib.suspended_refresh():
+        cmds.evalDeferred(
+            "from openpype.hosts.maya.api import lib;"
+            "lib.remove_render_layer_observer()")
+        cmds.evalDeferred(
+            "from openpype.hosts.maya.api import lib;"
+            "lib.add_render_layer_observer()")
+        cmds.evalDeferred(
+            "from openpype.hosts.maya.api import lib;"
+            "lib.add_render_layer_change_observer()")
```

Comment:
Dont think this is needed for this PR?

Reply:
Actually this should be removed because it's re-introducing legacy code that was removed on purpose.